### ISSUE TITLE
Decommission modules that were moved to core? token, pathauto ...

### ISSUE THREAD

**Issue by serundeputy** [backdrop/backdrop-issues#1096]
Should we as a matter of course decommission contrib modules that were move to core?

* https://github.com/backdrop-contrib/token
* https://github.com/backdrop-contrib/pathauto

etc.

**Comment by klonos** [comment]
I'm leaning towards voting "yes" for this, but we need to be specific about the tasks being involved in "decommissioning" a project...

Do we delete the entire github repo along with any issues and PR/commit history? Aren't these useful for posterity of what happened while porting from Drupal and then while being added to core?

So, perhaps a notification on the github repo (perhaps the README.md?) would be more appropriate I'm thinking.

**Comment by quicksketch** [comment]
Yeah I think just a (strong) note in the README.md would be sufficient to
decommission it.

On Mon, Aug 10, 2015 at 1:20 PM Gregory Netsas <notifications@github.com>
wrote:

> I'm leaning towards voting "yes" for this, but we need to be specific
> about the tasks being involved in "decommissioning" a project...
>
> Do we delete the entire github repo along with any issues and PR/commit
> history? Aren't these useful for posterity of what happened while porting
> from Drupal and then while being added to core?
>
> So, perhaps a notification on the github repo (perhaps the README.md?)
> would be more appropriate I'm thinking.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/backdrop/backdrop-issues/issues/1096#issuecomment-129594378>
> .
>
-- 

--
Sent from mobile

**Comment by Dinornis** [comment]
The Readme for the Token module now includes the line: `Important note: This project is included in core Backdrop 1.1.0, so you no longer need this module.`
Would be good to also include similar note in PathAuto.

**Comment by klonos** [comment]
@Dinornis 👍 ...I think I might be able to take care of that.

@quicksketch @serundeputy wasn't there some `requires` property that could be used in .info files of projects? We could set that to something like `backdrop < x.y.z` where xyz is the version of core when the module was merged in. That way, after an upgrade, the contrib modules would automatically get disabled and the respective core ones would get activated.
